Added translations for de, fr_FR, it, pt_PT

krga opened the pull request:
This PR has some translation additions for the following languages: de, fr_FR, it, pt_PT.

Thanks!

krga committed:
Added translations for de, fr_FR, it, pt_PT

krga commented:
missing mention also the translation for ro

JhonathanLepidus commented:
Hi! We're thankful for these contributions. They'll be very helpful for the community.

However, I noticed some issues with the translations you're sending. Usually, we use the same term PKP uses for "submission" on each language we translate, in order to maintain a cohesive user experience. Regarding this, I noticed the following locales don't follow this pattern:

- Italian: "Invii" instead of "Proposte"
- German: "Einreichung" instead of "Beitrag"
- Romenian: "Trimiterile" instead of "Înregistrare manuscris"
  - For this locale I also noticed it uses the term "Înscrieri" in other keys, which is not consistent.

Can you address these issues? We would be glad to merge this contribution when these issues are fixed.

krga committed:
reworked translations for de, it, ro according to PR feedback

krga commented:
Hi @JhonathanLepidus 

I've addressed your Feedback regarding the translations as good as possible. Please review the changes & decide if they should be included. Feedback is welcome.

Otherwise it would be helpful if these translation strings could be maintained in https://translate.pkp.sfu.ca. Are there any plans for this?

JhonathanLepidus commented:
Hi @krga 

Thanks for addressing the changes requested. I merged them, which are going to be available in the next release.

Regarding Weblate, we intend to integrate the translation of our plugins on it in a later moment, at least for the plugins available in the PKP's plugin gallery.
